scripts/node/icons/helpers/cleanExports.test.ts

```diff
@@ -0,0 +1,12 @@
+import { normalizePathToFile } from './cleanExports.js';
+
+describe('normalizing file names', () => {
+  test.concurrent.each([
+    ['/any/number/of/longer/paths/joeyTribbiani.png', 'JoeyTribbiani.png'],
+    ['/shorter/paths/monicageller.png', 'Monicageller.png'],
+    ['/any/path/CHANDLERBING.png', 'Chandlerbing.png'],
+    ['/the/path/gets/truncated/ross-gellar.png', 'RossGellar.png'],
+  ])('path of %s should return "%s"', (a, expected) => {
+    expect(normalizePathToFile(a)).toBe(expected);
+  });
+});
```

Comment:
I'm trying this style of testing for the first time. I don't know how I like it yet, but a couple of thoughts as to why I'm giving it a try out:

* Rather than type out a bunch of separate expectations and type more, I can make a clean abstraction. Someone can glance at the data, the parameters, and quickly understand how all the tests are related (first argument is input, 2nd argument is the expectation).
* The `path of %s should return %s%` bit, that will render the actual values of the array in the test name itself. So we don't lose context from the test.
* You may wonder why we are not using a forEach. One reason is potentially parallelization (not that this test matters, but as a pattern in general). The other reason is because the moment any test fails with a forEach, it will either stop running any additional tests in that file, or the test suite in general. This negatively impacts actual analytics of what is broken, and also breaks DX. If I go to fix one test and then the next one breaks, that's unideal.
* I also threw on jests `concurrent` mode (because why not, if it fails we just remove the word).

Reply:
amazing! i've always wanted this in my unit tests, this is something i was familiar with when i did unit testing in .net.
my vote is on this style, looks cleaner to me, and you write less code.